lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -140,7 +142,7 @@ createPendingRecord pool signedRecord = do
     -- TODO can I consolidate these two checks into one?
     -- check if a pending transaction already exists between the two users
     existingPending <- liftIO . withResource pool $ Db.lookupPendingByAddresses (creditor signedRecord) (debtor signedRecord)
-    unless (null existingPending) $
+    unless ( null existingPending ) $
```

Comment:
no spaces between parens and inner statement to be consistent with rest of the code.